components/html-block/html-block.js

```diff
@@ -255,8 +259,15 @@ class HtmlBlock extends LitElement {
 
 	_stamp(slot) {
 		const stampHTML = async template => {
-			const fragment = template ? document.importNode(template.content, true) : null;
-			if (fragment) {
+			let fragment;
+			if (template) fragment = document.importNode(template.content, true);
+			else {
+				fragment = document.createDocumentFragment();
+				this.shadowRoot.querySelector('slot').assignedNodes({ flatten: true })
+					.forEach(node => fragment.appendChild(node.cloneNode(true)));
```

Comment:
I think once we no longer support templates, it will make more sense to just skip constructing the fragment and just append these to the div directly (since the div won't yet be connected to the DOM anyways). However, for now, this makes the code branching a little less complicated right now, so I'm inclined to keep the fragment construction.